### ISSUE TITLE
pxe boot an image

### ISSUE THREAD

**Issue by hdt551** [Tomas-M/linux-live#53]
I am attempting to pxe boot an image. 
My "pxelinux.cfg/default" settings are:

**DEFAULT linux
LABEL linux
KERNEL /memdisk
APPEND iso initrd=linux-x86_64.iso raw**

The image boots, but eventually fails with
looking for linux data in /linux
Fatal error occured - could not locate linux data

Using a different iso image it will boot fine.
Is this because the iso created with linux-live is not "isohybrid"?
Is there another way I can boot my custom image?

**Comment by Tomas-M** [comment]
I don't fully understand what is the booting procedure. Your PXE client downloads the whole ISO file from your PXE server? What bootloader do you use on clients? You may need to use from=http://1.2.3.4/linux.iso append parametr to make it mount the ISO from network.

**Comment by hdt551** [comment]
Yes, the PXE client downloads the whole ISO file. I am not sure about the bootloader. I have the following files in the tftp download folder.
ldlinux.c32   libutil.c32       memdisk     pxelinux.cfg  
libcom32.c32  linux-x86_64.iso  pxelinux.0  slax.iso      vesamenu.c32
and as I said a different ISO file (in this case slax.iso) boots fine with this method.

I also tried booting my linux-live iso directly from CD and I get this output:
_switch root from initramfs to ramfs
Live Kit init <http://www.linux-live.org/>
Setting dynamic RAM compression using ZRAM if available
Probing for hardware
Looking for linux data in /linux ...
Found on dev/sr0
Setting up unions using AUFS
mount: mounting aufs on /memory/union failed: No such device
Adding bundles to union
01-core.sb
mount: mounting aufs on /memory/union failed: Invalid argument
/init: line 52: can't create /memory/union/etc/fstab: nonexistent directory
/init: line 52: can't create /memory/union/etc/fstab: nonexistent directory
/init: line 52: can't create /memory/union/etc/fstab: nonexistent directory
/init: line 52: can't create /memory/union/etc/fstab: nonexistent directory
/init: line 52: can't create /memory/union/etc/fstab: nonexistent directory
Live Kit done, starting linux
ln: sbin/fsck.aufs: No such file or directory
Fatal error occured - Can't find executable chroot command_


I must be doing something wrong but when I use the same files build from linux-live, copy them to a usb drive and run the build script, the usb drive will boot fine.  
There are no errors when I run the script to make the iso file.
